HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1480,174 +1705,7 @@ def self.apply_infiltration_and_ventilation_fans(model, weather, vent_mech, vent
-      # Consider fan rtf read in current calling point (results of previous time step) + CFIS_t_fan_on based on min/hr requirement and previous EMS results.
-      infil_program.addLine('    Set cfis_fan_runtime = @Max (@ABS(Minute - CFIS_t_fan_on)) (fan_rtf_hvac * ZoneTimeStep * 60)')
-      # If fan_rtf_hvac, make sure it's not exceeding ventilation requirements
-      infil_program.addLine("    Set cfis_fan_runtime = @Min cfis_fan_runtime (CFIS_t_min_hr_open - #{@cfis_t_sum_open_var.name})")
-      infil_program.addLine('    Set cfis_f_damper_open = cfis_fan_runtime/(60.0*ZoneTimeStep)') # calculates the portion of the current timestep the CFIS damper needs to be open
-      infil_program.addLine('    Set QWHV = cfis_f_damper_open*CFIS_Q_duct')
-      infil_program.addLine("    Set #{@cfis_t_sum_open_var.name} = #{@cfis_t_sum_open_var.name}+cfis_fan_runtime")
-      infil_program.addLine("    Set #{@cfis_f_damper_extra_open_var.name} = @Max (cfis_f_damper_open-fan_rtf_hvac) 0.0")
-      infil_program.addLine("    Set #{vent_mech_fan_actuator.name} = CFIS_fan_w*#{@cfis_f_damper_extra_open_var.name}")
-      infil_program.addLine('  Else')
-      # No need to turn on blower for extra ventilation
-      infil_program.addLine('    Set cfis_fan_runtime = fan_rtf_hvac*ZoneTimeStep*60')
-      infil_program.addLine("    If (#{@cfis_t_sum_open_var.name}+cfis_fan_runtime) > CFIS_t_min_hr_open")
-      # Damper is only open for a portion of this time step to achieve target minutes per hour
-      infil_program.addLine("      Set cfis_fan_runtime = CFIS_t_min_hr_open-#{@cfis_t_sum_open_var.name}")
-      infil_program.addLine('      Set cfis_f_damper_open = cfis_fan_runtime/(ZoneTimeStep*60)')
-      infil_program.addLine('      Set QWHV = cfis_f_damper_open*CFIS_Q_duct')
-      infil_program.addLine("      Set #{@cfis_t_sum_open_var.name} = CFIS_t_min_hr_open")
-      infil_program.addLine('    Else')
-      # Damper is open and using call for heat/cool to supply fresh air
-      infil_program.addLine('      Set cfis_fan_runtime = fan_rtf_hvac*ZoneTimeStep*60')
-      infil_program.addLine('      Set cfis_f_damper_open = fan_rtf_hvac')
-      infil_program.addLine('      Set QWHV = cfis_f_damper_open * CFIS_Q_duct')
-      infil_program.addLine("      Set #{@cfis_t_sum_open_var.name} = #{@cfis_t_sum_open_var.name}+cfis_fan_runtime")
-      infil_program.addLine('    EndIf')
-      # Fan power is metered under fan cooling and heating meters
-      infil_program.addLine("    Set #{vent_mech_fan_actuator.name} = 0")
-      infil_program.addLine('  EndIf')
-      infil_program.addLine('Else')
-      # The ventilation requirement for the hour has been met
-      infil_program.addLine('  Set QWHV = 0')
-      infil_program.addLine("  Set #{vent_mech_fan_actuator.name} = 0")
-      infil_program.addLine('EndIf')
-    else # Non-CFIS system
-      infil_program.addLine("Set QWHV = #{UnitConversions.convert(vent_mech_cfm.to_f, 'cfm', 'm^3/s').round(4)}")
```

Comment:
Small ci diff in erv/hrv systems is caused by removing the rounding here. Not a big issue applying rounding or not.